Composer/packages/server/src/models/utilities/util.ts

```diff
@@ -37,3 +37,7 @@ export const getRemoteFile = async (url): Promise<string> => {
   }
   return '';
 };
+
+export const convertFolderName = (path, folder) => {
```

Comment:
I dont understand what this util function is for, can you add types to the args, add better names to the args and function name, and add unit tests for it?

Reply:
Rename it & add test